test/framework/src/main/java/org/elasticsearch/test/junit/listeners/ReproduceInfoPrinter.java

```diff
@@ -181,10 +182,32 @@ private ReproduceErrorMessageBuilder appendESProperties() {
             appendOpt("tests.timezone", TimeZone.getDefault().getID());
             appendOpt("tests.distribution", System.getProperty("tests.distribution"));
             appendOpt("runtime.java", Integer.toString(JavaVersion.current().getVersion().get(0)));
+            appendOpt("runtime.java", Integer.toString(JavaVersion.current().getVersion().get(0)));
+            appendOpt("license.key", relativePath(System.getProperty("licence.key")));
             appendOpt(ESTestCase.FIPS_SYSPROP, System.getProperty(ESTestCase.FIPS_SYSPROP));
             return this;
         }
 
+        private String relativePath(String pathProperty) {
```

Comment:
Is this necessary once the fix is made in xpack to use Gradle's file resolution to get the path relative to the root project? With that change, and using a relative path in CI, shouldn't the reproduction line should just work for devs copying it from CI to their machines?

Reply:
This also works with having an absolute path passed in the commandline (e.g.on ci)

Reply:
But it will not required if we pass a relative path

Reply:
I'm just wondering if is necessary at all. It is specifically for CI. If a developer passes an absolute path, it will still work. We just need to ensure we pass a relative path in CI.

Reply:
in that case I can remove this

Reply:
done